@xen-orchestra/rest-api/src/rest-api/rest-api.type.mts

```diff
@@ -73,6 +73,7 @@ export type XoApp = {
   /* connect a server (XCP-ng/XenServer) */
   connectXenServer(id: XoServer['id']): Promise<void>
   /* disconnect a server (XCP-ng/XenServer) */
+  createGroup(params: { name: string }): Promise<XoGroup>
```

Comment:
Don't forget to add `provider` and `providerGroupId`.
Even if you are not using it on your `createGroup` endpoint, your type is here to document the function signature